fragdenstaat_de/theme/views.py

```diff
@@ -87,3 +92,32 @@ class FDSAnnotationView(AnnotateDocumentView):
     def get_queryset(self):
         qs = super().get_queryset()
         return qs.filter(portal__isnull=True)
+
+
+@login_required
+def scannerapp_postupload(request, message_pk):
```

Comment:
probably fine, but this means I can generate an (non-working) qr-code for any message_pk (even nonexistant ones), right?

Reply:
Yes, it's a bit dumb that way, but should be ok as long as the code does not actually access the `FoiMessage` and just passes the ID on.